tests/c/qsort.c

```diff
@@ -1,3 +1,11 @@
+// # This test breaks in swt tracer as swt tracer is missing unmappable block so it cannot
+// # see calls from unmappable blocks to mappable blocks and vice-versa. Disable this test
+// # for swt until we fix it.
+// # Example of what hwt see:
+// # mappable block (main):           <---
+// #        unmappable block (qsort)     |
+// #         mappable block (cmp)  ------
+// ignore-if: test "$YKB_TRACER" != "hwt"
```

Comment:
Better as  `ignore-if: test "$YKB_TRACER" == "swt"` (hwt and swt might not be the only tracers we have).

Reply:
This doesn't seem to work: `ignore-if: test "$YKB_TRACER" == "swt"`

Reply:
Try `=` (one equals sign).

Reply:
done!